backend/internal/service/handler/reviews/update_review.go

```diff
@@ -0,0 +1,102 @@
+package reviews
+
+import (
+	"github.com/gofiber/fiber/v2"
+	"platnm/internal/models"
+	"strconv"
+	"time"
+
+	"platnm/internal/errs"
+)
+
+type ReviewUpdate struct {
+	ID        int       `json:"id"`
+	UserID    string    `json:"user_id"`
+	Rating    *int      `json:"rating"`
+	Comment   *string   `json:"comment"`
+	UpdatedAt time.Time `json:"updated_at"`
```

Comment:
you can get away with not storing UpdatedAt in this struct, because your DB query can use postgres' `now()` function to automatically populate the updated_at time

Reply:
fixed!